docs/normalize-intervals.rst

```diff
@@ -0,0 +1,827 @@
+====================================
+
+.. sidebar:: The ISS Orbit
+
+    .. image:: _assets/img/normalize-intervals/orbit.png
+        :alt: An animated visualization of the ISS orbit
+        :target: https://en.wikipedia.org/wiki/International_Space_Station#/media/File:Animation_of_International_Space_Station_trajectory.gif
+
+    The ISS travels at 27,724 kilometers per hour and orbits Earth
+    approximately once every 90 minutes. Each complete wave on the graph
+    represents a full orbit. The amplitude oscillates
+
+If you followed one of the tutorials in the :ref:`previous section <gen-ts>`,
+you should have some mock time-series data about the position, or `ground
+point`_, of `International Space Station`_ (ISS).
+
+It is common to want to visualize time series data by graphing values over
+time. However, you may run into the following issues:
+
+1. The resolution of your data does not match the resolution you want for your
+   visualization.
+
+   *For example, you want to plot a single value per minute, but your data is
+   spaced in 10-second intervals. You will need to resample the data.*
+
+2. Your data is non-continuous, but you want to visualize a continuous time
+   series.
+
+   *For example, you want to plot every minute for the past 24 hours, but you
+   are missing data for some intervals. You will need to fill in the missing
+   values.*
+
+In this tutorial, we will demonstrate the shortcomings of visualizing the
+non-normalized data and show you how to address these shortcomings by
+normalizing your data using SQL.
+
+.. NOTE::
+
+    This tutorial focuses on the use of SQL. We have chosen to use a Python
+    client for demonstration purposes. However, *the following techniques will
+    work in any environment that allows for the execution of SQL*.
+
+.. SEEALSO::
+
+    :ref:`Tutorials for generating mock time-series data <gen-ts>`
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _ni-prereq:
+
+Prerequisites
+=============
+
+
+.. _ni-mock-data:
+
+Mock data
+---------
+
+You must have CrateDB :ref:`installed and running <install-run>`.
+
+This tutorial works with ISS location data. Before continuing, you should have
+generated some ISS data by following one of the tutorials in the :ref:`previous
+section <gen-ts>`.
+
+
+.. _ni-python:
+
+Python setup
+------------
+
+You should be using the latest stable version of `Python`_.
+
+We use the following Python libraries:
+
+- `pandas`_ -- querying and data manipulation
+- `SQLAlchemy`_ -- a database engine for pandas
+- The `CrateDB Python Client`_ -- CrateDB support for SQLAlchemy
+- `Matplotlib`_ -- data visualization
+
+You can install (or upgrade) the necessary libraries with `Pip`_:
+
+.. code-block:: console
+
+    sh$ pip install --upgrade pandas sqlalchemy crate matplotlib
+
+
+.. _ni-jupyter:
+
+Using Jupyter Notebook
+~~~~~~~~~~~~~~~~~~~~~~
+
+We have designed this tutorial to be run in `Jupyter Notebook`_ so that you can
```

Comment:
should be reworded to remove first person

Reply:
as above, I can't figure out how to do that without sacrificing the active voice :(

Reply:
went with this:

> This tutorial shows you how to use `Jupyter Notebook`_ so that you can display
data visually and experiment with the commands as you see fit.